src/asciidoc/docs/about/contribute.adoc

```diff
@@ -0,0 +1,133 @@
+
+The documentation is written in https://asciidoc.org/[Asciidoc], which is a lightweight markup language for simplifying the creation of documentation.
+All the AsciiDoc files that map to this site can be found in the repository under `src/asciidoc/`.
+
+Here's a quick example:
+
+[source, asciidoc]
+----
+= Koji for Developers
+
+Create your own templates for https://withkoji.com[Koji].
+
+== Use any framework
+
+* React
+* Vanilla JS
+
+[source,javascript]
+console.log('Hello World');
+----
+
+
+To see more examples and get a better understanding of the syntax, visit the https://powerman.name/doc/asciidoc[AsciiDoc cheatsheet] and the https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/[AsciiDoc Syntax Quick Reference].
+
+TIP: To test how your documentation changes will look on the site, clone the https://github.com/madewithkoji/docs[GitHub repository] locally and run `gatsby develop` in the root directory.
+
+We have also extended the AsciiDoc syntax to make the developer site ever better.
+
+=== Page attributes
+Page attributes are used to add extra data related to a document.
+Add these to the top of your document, just under the document title.
+
+Page slug::
+	Corresponds to the page URL and is used for linking between different pages (see <<#_documentation_structure>>). +
+	`:page-slug: example-page`
+Page description::
+	Provides a short description of the content of the page and is used when sharing the link on social media and chat platforms. +
+	`:page-description: This page is an example`
+Page banner::
+	References an image, which is used as the thumbnail when sharing the link on social media and chat platforms. +
+	`:page-banner: \{imagesDir\}/example-page.png`
+
+[NOTE]
+====
+- `\{imagesDir\}` refers to the image directory, which can be found in the repository at `static/images`
+- If the page contains block images, you don't need to set the page banner attribute. The first block image on the page is used, instead.
+====
+
+=== Tabs
+Sometimes it can be useful to sort your content into tabs.
+You'll see them used particularly often to display code snippets for differing frameworks.
+To add tabs into your document use the following syntax:
+
+[source, asciidoc]
+----
+include::../_includes/tabs-example.adoc[tags=*]
+----
+
+The above code will output the following:
+
+include::../_includes/tabs-example.adoc[tags=*]
+
+You will also notice that on some pages changing tabs will switch the tabs in multiple places (for example, choosing a coding framework in a blueprint article). 
+To sync multiple sets of tabs, use the `scope` attribute. Then, selecting a tab on the page also selects any other tabs with the same scope and a matching tab title.
+
+[source, asciidoc]
+----
+include::../_includes/tabs-scope-example.adoc[tags=*]
+----
+
+include::../_includes/tabs-scope-example.adoc[tags=*]
+
+=== Documentation structure
+Understanding the file structure for the documentation will also be helpful for making modifications.
```

Comment:
```suggestion
Understanding the file structure of the documentation is helpful when making modifications.
```
Just a style thing to avoid future tense ("will") when possible